### PR TITLE
Pull logger-loader out into `react-server-gulp-module-tagger`

### PR DESCRIPTION
**Pull request by gigabo**
TODO: Pass in the token to be replaced (e.g. `.pipe(tagger('__LOGGER__'))`)

### CODE REVIEW

File: packages/react-server-gulp-module-tagger/index.js [hunk 1]
```diff
@@ -6,10 +6,10 @@ var replace = require("gulp-replace")
 //   - "__LOGGER__({ /* options */ })"
 var isWindows = ('win32' === process.platform)
 ,   REPLACE_TOKEN = /(?:__LOGGER__|__CHANNEL__)(?:\(\s*(\{[\s\S]*?\})\s*\))?/g
-,   THIS_MODULE   = isWindows
-						? /(?:[^\\]+\\node_modules\\)?react-server\\buildutils\\logger-loader\.js$/
-						: /(?:[^\/]+\/node_modules\/)?react-server\/buildutils\/logger-loader\.js$/
 ,   BASE_PATH     = module.filename.replace(THIS_MODULE,'')
+,   THIS_MODULE   = isWindows
+	? /(?:[^\\]+\\node_modules\\)?react-server-gulp-module-tagger\.js$/
+	: /(?:[^\/]+\/node_modules\/)?react-server-gulp-module-tagger\.js$/
```
Comment (gigabo) [inline packages/react-server-gulp-module-tagger/index.js]: Oh, this is wrong.

**Comment by gigabo** [comment]
Okay, at long last.  Working.

**Comment by doug-wade** [comment]
:+1:

File: .travis.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ addons:
       - g++-4.8
 
 before_script:
-  - npm install -g gulp
-  - for d in `ls packages/`; do cd packages/$d; npm install; cd -; done
+  - npm install -g gulp lerna
+  - lerna bootstrap
```
Comment (roblg) [inline .travis.yml]: YES. I actually meant to leave this comment on one of the PRs from yesterday. `bootstrap` seems like the way it should be.

**Comment by roblg** [comment]
The names of these build plugins are outta control, but I think the diff looks reasonable. :+1:

File: .travis.yml (hunk 2 above)
Comment (gigabo) [inline .travis.yml]: Yeah, so much cleaner!

Incidentally I couldn't get lerna to link the two packages together without requiring `react-server-gulp-module-tagger` with a caret.  It [_looks_](https://github.com/kittens/lerna/blob/master/lib/commands/bootstrap/linkDependenciesForPackage.js#L48) like an exact version match should work.  I haven't traced it through to figure out why it didn't.

**Pull request merged**